examples/output.rs

```diff
@@ -0,0 +1,153 @@
+use bevy::prelude::*;
+use bevy_midi::output::*;
+
+const KEY_PORT_MAP: [(KeyCode, usize); 10] = [
+    (KeyCode::Key0, 0),
+    (KeyCode::Key1, 1),
+    (KeyCode::Key2, 2),
+    (KeyCode::Key3, 3),
+    (KeyCode::Key4, 4),
+    (KeyCode::Key5, 5),
+    (KeyCode::Key6, 6),
+    (KeyCode::Key7, 7),
+    (KeyCode::Key8, 8),
+    (KeyCode::Key9, 9),
+];
+
+const KEY_NOTE_MAP: [(KeyCode, u8); 7] = [
+    (KeyCode::A, 57),
```

Comment:
Any maybe this should be a feature instead of being part of the example

Reply:
The main reason I put the map of key codes to notes as a constant is to make as simple of an example as possible so it's easy to play with, even though it's a pretty dumb keyboard. I do think something like what you're suggesting could fit well in its own, more complex example though. 

I would argue that this shouldn't be a feature, because a keyboard like what you're describing would be really hard to build off of. Both UI and input could be handled very differently between two different apps. But if we made a separate example then people could just feel free to steal it and modify it as they please.

Reply:
Moved to #13